include/service/entities/site_query_service.js

```diff
@@ -260,13 +260,20 @@ module.exports = function SiteQueryServiceModule(pb) {
    * @param {Function} cb
    */
   SiteQueryService.prototype.getCollections = function (cb) {
-    this.listCollections({ name: { $nin: ["system.indexes", "system.namespaces"] }}, function(err, items) {
-      if(pb.util.isError(err)) {
-        pb.log.error(err);
-      }
-      cb(err, items);
-    });
-  };
+      this.listCollections({}, function(err, items) {
+          if(pb.util.isError(err)) {
+            pb.log.error(err);
+          }
+
+          for(var i = 0; i<items.length; i++) {
+              if(items[i].name.indexOf('system.indexes') != -1 || items[i].name.indexOf('system.namespaces') != -1){
+                items.splice(i, 1);
+              }
+          }
```

Comment:
In the event that you do need to filter them out at the web-server use the [Array.filter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter) function.  It will save a couple of lines.

```
items = items.filter(function(item) {
  return item.name.indexOf('system.indexes') === -1 && item.name.indexOf('system.namespaces') === -1;
});
```